source/documentation/test-scenarios.html.md.erb

```diff
@@ -30,14 +30,22 @@ When you are testing a declaration that requires a manual intervention, please s
   - the step number, for example, ‘step 3’, and action that you are expecting
   - the Local Reference Number (LRN)
 
-NCTS support staff monitor the [nctsp5tradertest@hmrc.gov.uk](mailto:nctsp5tradertest@hmrc.gov.uk) mailbox during 9am to 5pm Mondays to Fridays and release test declarations regularly. When we release your movement, we will reply to your email to let you know. If you have not received an update about your movement request after 1 working day, please send a follow-up email to the Trader Test team mailbox.
+NCTS support staff monitor the [nctsp5tradertest@hmrc.gov.uk](mailto:nctsp5tradertest@hmrc.gov.uk) mailbox during 9am to 5pm Mondays to Fridays and release test declarations regularly. When we release your movement, we will reply to your email to let you know. If you have not received an update about your movement request after 1 working day, please send a follow-up email to the declarant Test team mailbox.
 
 **Note:** The [nctsp5tradertest@hmrc.gov.uk](mailto:nctsp5tradertest@hmrc.gov.uk) mailbox must be used only to request manual intervention. It should not be used for general enquiries.
 
 **Note:** The automatic release period for scenarios using simplified departure procedure has been set to 45 minutes. This is to allow time for actions to be carried out where tests require manual intervention. In addition, the NCTS5 system reported time is 1 hour behind.
 
 **Important:** You must follow the requirements and instructions for each test precisely and you must provide movement-specific information if requested.
 
+### Message sender and recipient fields
+
+When submitting departure and arrival messages in Trader Test, you must comply with the following guidelines for entering message sender and recipient details.
+
+| Field | Guideline | Example |
+| ----- | --------- | ------- |
+| Message sender | Use your enrolment EORI number. | `<messageSender>9999912345</messageSender>` |
+| Message recipient | For departure messages, you must enter `NTA` and the correct code for the country of the office of departure. For example, if submitting a departure message for GB-XI, enter `NTA.GB`. Alternatively, if submitting a departure message for XI-GB, enter `NTA.XI`.<br /><br />For arrival messages, you must enter `NTA` and the correct code for the country of the office of destination. For example, if submitting an arrival message for GB-XI, enter `NTA.XI`. Alternatively, if submitting an arrival message for XI-GB, enter `NTA.GB`.<br /><br />**Note:** You should disregard the format and maximum length guidelines (‘an..35‘) in the NCTS5 technical interface specification. | `<messageRecipient>NTA.GB</messageRecipient>` |
```

Comment:
To be nitpicky, it's the office of destination actual, if an item is diverted that might be different from the office of destination declared. Do we want to make that distinction?

Otherwise, I was just looking at this PR to see if this was explained, which it is and is great. Is there somewhere else this should be specified outside of testing? Perhaps a UK specific rule in the TIS?

Reply:
Fair point. Will add.

Reply:
See 4th commit.